docs/site/content/en/docs/Reference/APIFilterQueryReference.md

```diff
@@ -0,0 +1,28 @@
+---
+title: Horreum API Filter Query Reference
```

Comment:
```suggestion
title: API Filter Queries
```

Reply:
The Title is used in the menu, "Horreum" and "Reference" are duplicated information in that context